idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/tests/test_phe.py

```diff
@@ -0,0 +1,173 @@
+                               },
+                               "cold_side":
+                               {
+                                   "property_package": m.fs.coldside_properties
+                               }})
+        return m
+
+    @pytest.fixture(scope="module", params=['dataset1', 'dataset2'])
+    def set_input_output(self, phe_model, measurement, request):
+        for t in phe_model.fs.time:
+            # hot fluid
+            Th_out = measurement[request.param]['output']['hotside']['temperature']
+            Th = measurement[request.param]['input']['hotside']['temperature']
+            Fh = measurement[request.param]['input']['hotside']['flowrate']
+            Ph = measurement[request.param]['input']['hotside']['pressure']
+            xh_CO2 = measurement[request.param]['input']['hotside']['mole_fraction']['CO2']
+            xh_MEA = measurement[request.param]['input']['hotside']['mole_fraction']['MEA']
+            xh_H2O = measurement[request.param]['input']['hotside']['mole_fraction']['H2O']
+
+            phe_model.fs.hx.hot_inlet.flow_mol[t].fix(Fh)
+            phe_model.fs.hx.hot_inlet.temperature[t].fix(Th)
+            phe_model.fs.hx.hot_inlet.pressure[t].fix(Ph)
+            phe_model.fs.hx.hot_inlet.mole_frac_comp[t, "CO2"].fix(xh_CO2)
+            phe_model.fs.hx.hot_inlet.mole_frac_comp[t, "H2O"].fix(xh_H2O)
+            phe_model.fs.hx.hot_inlet.mole_frac_comp[t, "MEA"].fix(xh_MEA)
+
+            # cold fluid
+            Tc_out = measurement[request.param]['output']['coldside']['temperature']
+            Tc = measurement[request.param]['input']['coldside']['temperature']
+            Fc = measurement[request.param]['input']['coldside']['flowrate']
+            Pc = measurement[request.param]['input']['coldside']['pressure']
+            xc_CO2 = measurement[request.param]['input']['coldside']['mole_fraction']['CO2']
+            xc_MEA = measurement[request.param]['input']['coldside']['mole_fraction']['MEA']
+            xc_H2O = measurement[request.param]['input']['coldside']['mole_fraction']['H2O']
+            phe_model.fs.hx.cold_inlet.flow_mol[t].fix(Fc)
+            phe_model.fs.hx.cold_inlet.temperature[t].fix(Tc)
+            phe_model.fs.hx.cold_inlet.pressure[t].fix(Pc)
+            phe_model.fs.hx.cold_inlet.mole_frac_comp[t, "CO2"].fix(xc_CO2)
+            phe_model.fs.hx.cold_inlet.mole_frac_comp[t, "H2O"].fix(xc_H2O)
+            phe_model.fs.hx.cold_inlet.mole_frac_comp[t, "MEA"].fix(xc_MEA)
+
+            output = {'hotside_temperature_expectation': Th_out,
+                      'coldside_temperature_expectation': Tc_out}
+            return output
+
+    @pytest.mark.unit
+    def test_phe_build(self, phe_model):
+        assert phe_model.fs.hx.config.dynamic is False
+        assert degrees_of_freedom(phe_model) == 12
+
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.integration
```

Comment:
This one actually looks like an integration test, so this mark is probably correct.